### PR TITLE
Logs panel: Add option to show common labels

### LINKED ISSUE #20449: Log panel: Display also common labels
**What would you like to be added**:

I would love to see labels in the logs panel, as shown on explore, as well as common labels.

### PR DESCRIPTION
**Pull request by ivanahuckova**
**What this PR does / why we need it**:
This PR adds option to show common labels in Logs panel. 

image
image

**Which issue(s) this PR fixes**:

Fixes https://github.com/grafana/grafana/issues/20449

### CODE REVIEW

**Comment by oddlittlebird** [comment]
Doc change looks fine.

**Review by achatterjee-grafana — APPROVED** [review]

**Comment by achatterjee-grafana** [comment]
Does this need backporting to 8.0.x doc set?

**Comment by ivanahuckova** [comment]
> Does this need backporting to 8.0.x doc set?

No, it would be in 8.1

File: public/app/plugins/panel/logs/LogsPanel.tsx [hunk 1]
```diff
@@ -39,15 +41,17 @@ export const LogsPanel: React.FunctionComponent<LogsPanelProps> = ({
...
-
   return (
     <CustomScrollbar autoHide>
-      <div className={spacing}>
+      <div className={style.container}>
+        {showCommonLabels && (
+          <div className={style.labelContainer}>
+            <span className={style.label}>Common labels:</span>
+            <LogLabels
+              labels={commonLabels ? (commonLabels.value as Labels) : ({ labels: '(no common labels)' } as Labels)}
```
Comment (gabor) [inline public/app/plugins/panel/logs/LogsPanel.tsx:51]:
```suggestion
              labels={commonLabels ? (commonLabels.value as Labels) : ({ labels: '(no common labels)' })}
```
```suggestion
              labels={commonLabels ? (commonLabels.value as Labels) : ({ labels: '(no common labels)' } as Labels)}
```

i think the second `as Labels` is not necessary (well, i removed it and typescript did not complain)

**Review by gabor — APPROVED** [review]
looks good to me! (i left one small nitpick)

**Comment by ivanahuckova** [comment]
@gabor Thanks! I have committed your change!

**Pull request merged**